release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -484,7 +484,7 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-se
 [id="ocp-4-14-egress-ips-additional-networks"]
 ==== Support for egress IPs on additional network interfaces (Technology Preview)
 
-Support for egress IPs on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements. 
+Support for egress IPs on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements.
```

Comment:
```suggestion
Support for egress IP addresses on additional network interfaces is now available as a Technology Preview feature. This feature provides {product-title} administrators with a greater level of control over networking aspects such as routing, addressing, segmentation, and security policies. It also provides users with the option to route workload traffic over specific network interfaces for purposes such as traffic segmentation or meeting specialized requirements.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

Reply:
I'll grab this, but this was just eliminating the trailing whitespace.